spec/latest/core/index.html

```diff
@@ -601,6 +601,14 @@ <h3>Documenting a Web API</h3>
       for the operation invocation.
     </pre>
 
+    <p>To wrap up everything altogether, it is also possible to attach atomic
+      operations supported by, well, supported property itself. This might
+      come in handy for scenarios, when resource can be partially modified.
```

Comment:
This is a little unclear without an example TBH. 

I used this feature for example on a "typed" collection link.

```turtle
# in data
<> a api:Entrypoint ; api:dataSheetCollection <data-sheets> .

# in API Documentation
api:Entrypoint
  hydra:supportedProperty [
    hydra:property api:dataSheetCollection ;
  ] .

api:dataSheetCollection
  a                        hydra:Link ;
  rdfs:range               <api#DataSheetCollection> ;
  hydra:supportedOperation _:PostToCollection .
```

This way my client dereferences `<>` (entrypoint) and discovers that it can POST to `<data-sheets>`.

----

So now that would change like so: 

```diff
api:Entrypoint
  hydra:supportedProperty [
    hydra:property api:dataSheetCollection ;
+   hydra:supportedOperation _:PostToCollection ;
  ] .

-api:dataSheetCollection
-  a                        hydra:Link ;
-  rdfs:range               <api#DataSheetCollection> ;
-  hydra:supportedOperation _:PostToCollection .
```

Reply:
Good point. I'll make required amendments to over that scenario.
In general, sole purpose was to make examples like `<Me> schema:knows <You>` obsolete as `schema:knows` is not `hydra:Link`. But your example clearly states that `api:dataSheetCollection` is a `hydra:Link`, thus it can have a `hydra:supportedProperty` attached.

Reply:
Done. I was wondering whether providing both examples could be useful.